NorthstarDedicatedTest/masterserver.cpp

```diff
@@ -1109,14 +1117,15 @@ void MasterServerManager::WritePlayerPersistentData(char* playerId, char* pdata,
 		{
 			CURL* curl = curl_easy_init();
 			SetCommonHttpClientOptions(curl);
+			curl_easy_setopt(curl, CURLOPT_USERAGENT, &NSUserAgent);
```

Comment:
Why is this not being done in `SetCommonHttpClientOptions`?

Reply:
Resolved in 3643deb64efe15ab38af27bd605b03d587cf1d5b

Reply:
Resolved in 34da460021e723ef09e45fecd9e65018524fa1d7